website/docs/r/evidently_launch.html.markdown

```diff
@@ -0,0 +1,349 @@
+* `segment_overrides` - (Required) One or up to six blocks that specify different traffic splits for one or more audience segments. A segment is a portion of your audience that share one or more characteristics. Examples could be Chrome browser users, users in Europe, or Firefox browser users in Europe who also fit other criteria that your application collects, such as age. [Detailed below](#segment_overrides).
+* `start_time` - (Required) Specifies the date and time that this step of the launch starts.
+
+##### `segment_overrides`
+
+* `evaluation_order` - (Required) Specifies a number indicating the order to use to evaluate segment overrides, if there are more than one. Segment overrides with lower numbers are evaluated first.
+* `segment` - (Required) The name or ARN of the segment to use.
+* `weights` - (Required) The traffic allocation percentages among the feature variations to assign to this segment. This is a set of key-value pairs. The keys are variation names. The values represent the amount of traffic to allocate to that variation for this segment. This is expressed in thousandths of a percent, so a weight of 50000 represents 50% of traffic.
+
+## Timeouts
+
+[Configuration options](https://www.terraform.io/docs/configuration/blocks/resources/syntax.html#operation-timeouts):
+
+* `create` - (Default `2m`)
+* `delete` - (Default `2m`)
+* `update` - (Default `2m`)
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `arn` - The ARN of the launch.
+* `created_time` - The date and time that the launch is created.
+* `execution` - A block that contains information about the start and end times of the launch. [Detailed below](#execution)
+* `id` - The launch `name` and the project `name` or `arn` separated by a colon (`:`).
+* `last_updated_time` - The date and time that the launch was most recently updated.
+* `status` - The current state of the launch. Valid values are `CREATED`, `UPDATING`, `RUNNING`, `COMPLETED`, and `CANCELLED`.
+* `status_reason` - If the launch was stopped, this is the string that was entered by the person who stopped the launch, to explain why it was stopped.
+* `tags_all` - A map of tags assigned to the resource, including those inherited from the provider [`default_tags` configuration block](/docs/providers/aws/index.html#default_tags-configuration-block).
+* `type` - The type of launch.
+
+### `execution`
+
+The `execution` block supports the following attributes:
+
+* `ended_time` - The date and time that the launch ended.
+* `started_time` - The date and time that the launch started.
+
+## Import
+
+CloudWatch Evidently Launch can be imported using the launch `name` and `name` or `arn` of the hosting CloudWatch Evidently Project separated by a `:`, e.g.,
```

Comment:
This is a little unclear, whats the diff b/w the launch name and name? perhaps add an example of each way to import